### PR TITLE
complete 05

### PR DESCRIPTION
**Pull request by moskgit**
Complete ticket 05
- create description adding script in a separate file to keep it neat.
- create 2 text files called queries and endPointsList for reference

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -67,24 +67,72 @@ describe("GET /api/articles/:article_id", () => {
...
+            author: 'butter_bridge',
+            created_at: '2020-07-09T20:11:00.000Z',
+            votes: 100,
+            article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700',
+            comment_count: '11'
+          });
+        }
+      });
+      });
+  });
```
Comment (RichardNeat) [inline __tests__/app.test.js:130]: We also need to check the articles are sorted by date in descending order. You may want to look into jest sorted to help you test for this - https://www.npmjs.com/package/jest-sorted

File: app.js [hunk 2]
```diff
@@ -15,6 +15,12 @@ app.get('/api/', getApiEndPoints)
 
 app.get('/api/articles/:article_id', getArticlesById);
 
+app.get('/api/articles', getAllArticles);
+
+app.all('/api/*', (req, res, next) => {
```
Comment (RichardNeat) [inline app.js]: Nice work having this generic bad path handler in here. What if I navigate to the endpoint /banana? A bad path may not just start with /api so best to simply pass '*' into here to grab all bad paths.

File: queries [hunk 3]
```diff
@@ -0,0 +1,30 @@
+--This is just a spare file.
```
Comment (RichardNeat) [inline queries:1]: Not a problem at all to have a sandbox file to play around with, but ideally we shouldn't be committing this up to be included in our production code :)

**Review by RichardNeat — CHANGES_REQUESTED** [review]
Nice work! Just a few requested changes below please - once those changes are done you can go ahead and merge. If you have any questions drop me a message on slack (Rich-NC) 😄

File: __tests__/app.test.js (hunk 1 above)
Comment (moskgit) [reply __tests__/app.test.js:130]: Added the test.

File: app.js (hunk 2 above)
Comment (moskgit) [reply app.js]: removed /api/ that leaves * only.

File: queries (hunk 3 above)
Comment (moskgit) [reply queries:1]: Can I keep the files in the git hub for now and delete them at the final stage? don't want to keep tracking them locally.

**Comment by moskgit** [comment]
Refactored

**Review by robert112233 — APPROVED** [review]
Nice work, merge!

**Pull request merged**